.github/workflows/build.yml

```
@@ -39,18 +39,37 @@ jobs:
         with:
           version: "17.0"
 
+      - name: Install Conan
+        id: conan
+        uses: turtlebrowser/get-conan@main
+
+      - name: Setup Conan Packages
+        shell: bash
+        run: |
+          conan profile detect --force
+          conan install . --settings compiler.cppstd=17 -s build_type=$BUILD_TYPE --build=missing
+          cd build
+          ls
+          cd $BUILD_TYPE
+          ls
+          cd generators
+          ls
+          cd ..
+          cd ..
+          cd ..
+
       - name: Install CMake
         uses: lukka/get-cmake@latest
         
       - name: Configure
         shell: bash
         run: |
           if [ "${{ env.COMPILER }}" == "msvc" ]; then
-            cmake -S . -B build -DCMAKE_BUILD_TYPE=$BUILD_TYPE -G "Visual Studio 17 2022" -DCMAKE_CXX_COMPILER_TARGET=x86_64-pc-$OPERATING_SYSTEM-msvc -DCMAKE_C_COMPILER_TARGET=x86_64-pc-$OPERATING_SYSTEM-msvc
+            cmake -S . -B build -DCMAKE_BUILD_TYPE=$BUILD_TYPE -G "Visual Studio 17 2022" -DCMAKE_CXX_COMPILER_TARGET=x86_64-pc-$OPERATING_SYSTEM-msvc -DCMAKE_C_COMPILER_TARGET=x86_64-pc-$OPERATING_SYSTEM-msvc -DCMAKE_MODULE_PATH=build/$BUILD_TYPE/generators -DCMAKE_PREFIX_PATH=build/$BUILD_TYPE/generators
```

```suggestion
            cmake -S . -B build -DCMAKE_BUILD_TYPE=$BUILD_TYPE -G "Visual Studio 17 2022" -DCMAKE_CXX_COMPILER_TARGET=x86_64-pc-$OPERATING_SYSTEM-msvc -DCMAKE_C_COMPILER_TARGET=x86_64-pc-$OPERATING_SYSTEM-msvc -DCMAKE_TOOLCHAIN_FILE=build/$BUILD_TYPE/generators/conan_toolchain.cmake
```
This is the way Conan expects the generated dependencies to be used typically. Are you specifically trying to avoid this approach for some reason?

---

Ok, makes sense. That's annoying indeed.

Another option is to use the generated CMake presets instead.
See https://github.com/valgur/velodyne_decoder/blob/master/.github/workflows/build.yml#L101-L149 for an example.

Alternatively, you can let the official conan_provider.cmake module (that I've made use of in my PR) take care of creating a matching Conan configuration, installation and use of Conan dependencies automatically.